src/main/kotlin/wordle/view/OutputView.kt

```diff
@@ -0,0 +1,24 @@
+package wordle.view
+
+import wordle.domain.Game
+import wordle.domain.Tile
+
+object OutputView {
+
+    fun printStartMessage() = println(
+        "WORDLE을 ${Game.MAX_GAME_COUNT}번 만에 맞춰 보세요.\n" + "시도의 결과는 타일의 색 변화로 나타납니다."
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        "WORDLE을 ${Game.MAX_GAME_COUNT}번 만에 맞춰 보세요.\n시도의 결과는 타일의 색 변화로 나타납니다."
```